utils/gen_onnx_mlir.py

```diff
@@ -47,193 +47,193 @@
 # run this script with --check-operation-version flag.
 # Update this dictionary when a newer version is implemented
 # TODO: how to keep the old version
```

Comment:
Yes, it may be a good place to add what to do when you want to keep the old ones.
Also, what do you recommend when there is an other operation, should it be -1... or some value that indicate "not supported?" Or not being there indicate not supported?

Reply:
When searching for operation with version (in FrontendDialectTransformer.cpp), it is assumed that all versions are supported. If the version is not directly supported, the closest higher version is used. For example, if version 1, 3 and 5 are supported, version 3 is used for version 2 op in model. Can add an array for not supported op+version.